core/css.js

```diff
@@ -142,12 +142,13 @@ Blockly.Css.CONTENT = [
   '  background-color: #fff;',
   '  outline: none;',
   '  overflow: hidden;',  /* IE overflows by default. */
+  '  display: block;', 
   '}',
 
   '.blocklyWidgetDiv {',
   '  display: none;',
   '  position: absolute;',
-  '  z-index: 999;',
+  '  z-index: 99999 !important;',
```

Comment:
Is the `!important` really necessary? It makes overwriting the css property in your own css file not really possible.